profile_bluesky/startup/instrument/devices/ad_imm_plugins.py

```diff
@@ -95,7 +97,7 @@ def setIMM_Raw(self):
         Set all IMM plugins for raw (uncompressed).
         """
         # from SPEC macro: ccdset_RawMode_params_ad_Lambda
-        for plugin in (self.imm1, self.imm2, self.immout):
+        for plugin in (self.imm1):
             if plugin.file_format.get() not in (0, "IMM_Raw"):
```

Comment:
same as above, no `file_format` attribute now - what to do here?

Reply:
Same as above